packages/flutter/lib/src/widgets/scroll_view.dart

```diff
@@ -430,6 +430,68 @@ abstract class ScrollView extends StatelessWidget {
 /// ```
 /// {@end-tool}
 ///
+/// {@tool dartpad --template=stateful_widget_material}
+///
+/// This sample code shows a scroll view that contains two [SliverList],
+/// allowing the scroll view to grow in both directions, by setting the
+/// [CustomScrollView.center] to the key of the second list.
+///
+/// ```dart
+/// List<int> top = [];
+/// List<int> bottom = [0];
+///
+/// @override
+/// Widget build(BuildContext context) {
+///   return Scaffold(
+///     appBar: AppBar(
+///       title: const Text('Press on the plus to add items above and below'),
+///       leading: IconButton(
+///         icon: const Icon(Icons.add),
+///         onPressed: () {
+///           setState(() {
+///             top.add(-top.length - 1);
+///             bottom.add(bottom.length);
+///           });
+///         },
+///       ),
+///     ),
+///     body: CustomScrollView(
+///       center: const ValueKey('second-sliver-list'),
```

Comment:
I recommend extracting the key to a variable so you don't duplicate it below

Reply:
Good point, done!